cypress/e2e/v1.0/accommodation/questions/wantToMakeChanges.ts

```diff
@@ -0,0 +1,91 @@
+export default (stepUrl: string, summaryPage: string, positionNumber: number, totalQuestionsOnPage?: number) => {
+  const question = 'Does Sam want to make changes to their accommodation?'
+  const options = [
+    'I have already made positive changes and want to maintain them',
+    'I am actively making changes',
+    'I want to make changes and know how to',
+    'I want to make changes but need help',
+    'I am thinking about making changes',
+    'I do not want to make changes',
+    'I do not want to answer',
+    null,
+    'Sam is not present',
+    'Not applicable',
+  ]
+
+  describe(question, () => {
+    it(`displays and validates the question`, () => {
+      cy.getQuestion(question)
+        .isQuestionNumber(positionNumber)
+        .hasHint('This question must be directly answered by Sam.')
+        .hasRadios(options)
+      cy.saveAndContinue()
+      cy.assertStepUrlIs(stepUrl)
+      cy.getQuestion(question).hasValidationError('Select if they want to make changes to their accommodation')
+    })
+    ;[
+      'I have already made positive changes and want to maintain them',
+      'I am actively making changes',
+      'I want to make changes and know how to',
+      'I want to make changes but need help',
+      'I am thinking about making changes',
+      'I do not want to make changes',
+    ].forEach(option => {
+      it(`conditional field is displayed for "${option}"`, () => {
+        cy.getQuestion(question).getRadio(option).hasHint(null).hasConditionalQuestion(false).clickLabel()
+
+        cy.getQuestion(question)
+          .getRadio(option)
+          .getConditionalQuestion()
+          .hasTitle('Give details (optional)')
+          .hasHint(null)
+          .hasLimit(400)
+
+        cy.saveAndContinue()
+
+        // Move us back to the step when this is the only question on the page, as a save will continue to the next step
```

Comment:
Interesting problem 🤔 technically we could have multiple optional questions on a page and still go to the next step.

what about `if(currentUrl !== stepUrl) cy.visit(stepUrl)` in tests where we don't expect validation errors? 🤔